src/targets/registry.ts

```diff
@@ -396,78 +423,88 @@ export class RegistryTarget extends BaseTarget {
     return git;
   }
 
+  public async getValidItems(
```

Comment:
Could be `private`?

Reply:
Eh?

Reply:
This could be a private method, yes? Since it's only used from `this.publish`?

Reply:
Yes it can be. I just don't really see a reason to discuss/change this. Making this public would make testing easier.

Reply:
> would make testing easier

I _did_ notice a lack of tests. 😜

Reply:
Hopefully there will be. For now we just rely on the extensive manual testing that I did. There were no tests anyway so I'm not leaving this place worse than I found it? 😝